src/main/resources/application.properties

```diff
@@ -71,6 +71,13 @@ galapagos.kafka.certificates-workdir=file:/tmp
 # The prefix for Galapagos internal topics
 galapagos.kafka.metadataTopicsPrefix=galapagos.internal.
 
+# entries specifies the minimum number of changes in the dashboard.
+# minDays indicates that all changes since X days ago are displayed in the dashboard. The bigger value gets used.
+# Example: Entries is set to 10, minDays is set to 30. In the last 30 days were 28 Changes.
+# The dashboard will show 28 changes.
```

Comment:
Please add a note that this is a UI-only configuration and has no effect on the REST endpoint for retrieving the changelog.